krun/platform.py

```diff
@@ -654,8 +654,16 @@ def __init__(self, mailer, config):
         self.temp_sensor_map = None
         UnixLikePlatform.__init__(self, mailer, config)
         self.num_cpus = self._get_num_cpus()
+        self.num_per_core_measurements = self._get_num_per_core_measurements()
+
         self.virt_what_cmd = None  # set later
 
+    def _get_num_per_core_measurements(self):
+        # For all systems apart from travis we expect per-core measurements
+        if os.environ.get("TRAVIS") == "true":
```

Comment:
Note -- if this kind of thing happens much more (I hope not), we should make a `class TravisPlatform(DebianLinuxPlatform)`. Not now though.